config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
+end
```

Comment:
[REQUIRED] Kindly include `new` and `create` routes for posts and comments

[REQUIRED] Kindly include `create` action for likes